lms/djangoapps/homepage/views.py

```diff
@@ -18,6 +20,15 @@ def home_page(request, user=AnonymousUser()):
 
 
 def get_featured_courses(user):
+    featured_courses = []
+
     courses = get_courses(user)
     featured_custom_settings = CustomSettings.objects.filter(is_featured=True).values_list('id', flat=True)
-    return [course for course in courses if str(course.id) in featured_custom_settings]
+    for course in courses:
+        if str(course.id) in featured_custom_settings:
+            course_details = modulestore().get_course(course.id)
+            instructors = course_details.instructor_info.get('instructors')
+            course.instructor = instructors[0] if instructors else {}
```

Comment:
It is quite possible a course can have multiple instructors. I think we should pass all instructors and let front end decides how it can display instructor info.